services/user/types.go

```diff
@@ -30,3 +30,9 @@ type Login struct {
 type PaypalMailAddrSetter struct {
 	PaypalMailAddr string `json:"paypal_mail_addr"`
 }
+
+// PublicUser is only for userdata which every other user can see.
+type PublicUser struct {
```

Comment:
Shouldn't we name this `Profile` or something? I guess `user.Profile` sounds better than `user.PublicUser`

Reply:
yes, but a Profile could also be from viewpoint of the user itself. It does not imply that it's public... maybe user.PublicProfile
Later we may have a PrivateProfile...